vendor/k8s.io/kubernetes/pkg/kubectl/cmd/delete.go

```diff
@@ -252,7 +254,23 @@ func (o *DeleteOptions) DeleteResult(r *resource.Result) error {
 		}
 		options.PropagationPolicy = &policy
 
-		return o.deleteResource(info, options)
+		response, err := o.deleteResource(info, options)
+		if err != nil {
+			return err
+		}
+		if status, ok := response.(*metav1.Status); ok && status.Details != nil {
+			uidMap[kubectlwait.ResourceLocation{Namespace: info.Namespace, Name: info.Name}] = status.Details.UID
```

Comment:
does delete never handle multiple resource types? I expected at least group/resource in the key as well

Reply:
> does delete never handle multiple resource types? I expected at least group/resource in the key as well

oops.  Yep.

Reply:
> does delete never handle multiple resource types? I expected at least group/resource in the key as well

fixed and test updated